extensions/positron-assistant/src/md/prompts/chat/default.md

```diff
@@ -35,3 +112,647 @@ results, generate the code and return it directly without trying to execute it.
+import polars as pl
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# Prepare data for plotting from the base DataFrame
+df_plot_bar = df.group_by("city").agg(pl.col("score").mean().alias("avg_score"))
+
+# Convert Polars DataFrame to Pandas DataFrame for Seaborn compatibility.
+plt.figure(figsize=(10, 6))
+sns.barplot(x="city", y="avg_score", data=df_plot_bar.to_pandas())
+plt.title("Average Score by City")
+plt.xlabel("City")
+plt.ylabel("Average Score")
+plt.show()
+```
+
+## Seaborn Box Plot
+
+```python
+import polars as pl
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# Prepare data for plotting from the base DataFrame
+df_plot_box = df.select(pl.col("category"), pl.col("value"))
+
+# Convert Polars DataFrame to Pandas DataFrame for Seaborn compatibility.
+plt.figure(figsize=(8, 6))
+sns.boxplot(x="category", y="value", data=df_plot_box.to_pandas())
+plt.title("Distribution of Value by Category")
+plt.xlabel("Category")
+plt.ylabel("Value")
+plt.show()
+```
+
+## Seaborn Scatter Plot
+
+```python
+import polars as pl
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+
+# Create dummy data for scatter plot
+df_scatter = pl.DataFrame({
+    "x_data": np.random.rand(50) * 100,
+    "y_data": np.random.rand(50) * 50,
+    "group": np.random.choice(["Group1", "Group2"], 50),
+})
+
+# Convert Polars DataFrame to Pandas DataFrame for Seaborn compatibility.
+plt.figure(figsize=(9, 7))
+sns.scatterplot(x="x_data", y="y_data", hue="group", data=df_scatter.to_pandas())
+plt.title("Scatter Plot of X vs Y by Group")
+plt.xlabel("X Data")
+plt.ylabel("Y Data")
+plt.show()
+```
+
+
+## Plotnine: Scatter Plot with Polars DataFrame
+
+```python
+import polars as pl
+from plotnine import ggplot, aes, geom_point, geom_bar, labs, theme_minimal
+import numpy as np
+
+df_scatter_plotnine = pl.DataFrame(
+    {
+        "x_coord": np.random.rand(50) * 10,
+        "y_coord": np.random.rand(50) * 10,
+        "category": np.random.choice(["Group A", "Group B", "Group C"], 50),
+    }
+)
+
+p_scatter = (
+    ggplot(df_scatter_plotnine, aes(x="x_coord", y="y_coord", color="category"))
+    + geom_point(size=3, alpha=0.7)
+    + labs(
+        title="Plotnine Scatter Plot",
+        x="X-coordinate",
+        y="Y-coordinate",
+        color="Category"
+    )
+    + theme_minimal() # Using theme_minimal for a clean look
+)
+p_scatter.show()
+```
+
+## Plotnine: Bar Plot with Polars DataFrame
+
+```python
+df_bar_plotnine = pl.DataFrame(
+    {
+        "item": ["Apple", "Banana", "Cherry", "Date"],
+        "count": [15, 22, 10, 18],
+    }
+)
+
+p_bar = (
+    ggplot(df_bar_plotnine, aes(x="item", y="count", fill="item"))
+    + geom_bar(stat="identity") # stat="identity" means the y-values are actual counts
+    + labs(
+        title="Plotnine Bar Plot",
+        x="Item",
+        y="Count"
+    )
+    + theme_minimal()
+)
+p_bar.show()
+```
+</examples-python>
+
+The next messages you see will be from the USER.
```

Comment:
Is this true? There are a lot of places where we concatenate the default prompt with a more specific one. Do we want to save this statement for the end of the concatenated prompt?

Reply:
Ah yes, you are right. IIRC we also sometimes add extra messages before the real user messages.

I'm going to remove this line for now. I am not fully convinced by its efficacy in any case, despite it being used in similar code assistant prompts.

We can always add something like it back in a future PR, once we have an eval framework and we can show it makes a measurable difference.